dev/vagrant/conf/consul/README.md

```diff
@@ -1,8 +1,5 @@
-# Overriding and appending consul configuration
+# Consul configuration
+You may add any `.hcl` or `.json` -files to this directory in order to change or override the configuration.
+Any valid [consul configuration](https://www.consul.io/docs/agent/options.html#configuration_files) added to this directory will append the configuration, in lexical order. 
 
-You may add any hcl-files to this directory in order to change the configuration.
-Any valid configuration added to this directory will append the configuration, in lexical order.
-
-Adding a file `99-override.hcl` you will ensure it will be appended last, and 00-override.hcl will be read first.
-Any valid configuration from [https://www.consul.io/docs/agent/options.html#configuration_files](https://www.consul.io/docs/agent/options.html#configuration_files) will work.
-See [example](../../../../template_example/dev/vagrant/conf/consul/99-override.hcl)
+A example file is provided in [template_example/dev/vagrant/conf/consul](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/template_example/dev/vagrant/conf/consul).
```

Comment:
```suggestion
An example file is provided in [template_example/dev/vagrant/conf/consul](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/template_example/dev/vagrant/conf/consul).
It means that by prefixing a filename with `00` (e.g. `00-override.hcl`), it will be read first and using `99` (e.g. `99-override.hcl`) it will be read last.
```